### PR TITLE
fix: spelling error in footer

### PR DESCRIPTION
**Pull request by ksba2004**
# Pull Request Template

## Description

I have made changes in "Working Groups" spelling present in the footer section, along with it I have also fixed the Nav Bar to top and have made the text-alignment to left.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).

## How Has This Been Tested?

These changes are tested on Windows 11 Google Chrome and Microsoft Edge Platform.

## Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added corresponding tests.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

### CODE REVIEW

File: website/src/components/HomePageFeatures/HomepageFeatures.module.css [hunk 1]
```diff
@@ -9,7 +9,7 @@
 }
 
 .featureImage {
-  text-align: center;
+  text-align:center;
```
Comment (padmajabhol) [inline website/src/components/HomePageFeatures/HomepageFeatures.module.css:12]: please revert this change

File: website/src/components/ContributeCard/styles.css [hunk 2]
```diff
@@ -1,6 +1,6 @@
 .card-container{
     display: flex;
-    justify-content: center;
+    justify-content:left;
```
Comment (padmajabhol) [inline website/src/components/ContributeCard/styles.css:3]: is this exported anywhere?

File: website/src/components/HomePageFeatures/HomepageFeatures.module.css [hunk 3]
```diff
@@ -38,6 +38,7 @@
 
 .featureDescription {
   font-size: medium;
+  text-align: left;
```
Comment (padmajabhol) [inline website/src/components/HomePageFeatures/HomepageFeatures.module.css:41]: I dont think it makes any difference?

**Review by padmajabhol — CHANGES_REQUESTED** [review]

File: website/src/components/HomePageFeatures/HomepageFeatures.module.css (hunk 1 above)
Comment (ksba2004) [reply website/src/components/HomePageFeatures/HomepageFeatures.module.css]: i have reverted this change, please check the commit

File: website/src/components/HomePageFeatures/HomepageFeatures.module.css (hunk 3 above)
Comment (ksba2004) [reply website/src/components/HomePageFeatures/HomepageFeatures.module.css:41]: it makes changes in "Engage with Community" section, earlier the text was aligned center and now it is aligned to left.

File: website/src/components/ContributeCard/styles.css (hunk 2 above)
Comment (ksba2004) [reply website/src/components/ContributeCard/styles.css:3]: no, it hasn't been exported anywhere.

Comment (padmajabhol) [reply website/src/components/ContributeCard/styles.css:3]: please revert back the changes for now

File: website/src/components/HomePageFeatures/HomepageFeatures.module.css (hunk 3 above)
Comment (padmajabhol) [reply website/src/components/HomePageFeatures/HomepageFeatures.module.css:41]: Revert back the changes.

**Review by HarshCasper — APPROVED** [review]
